easybuild/easyblocks/h/hypre.py

```diff
@@ -40,20 +42,20 @@ def configure_step(self):
 
         self.cfg.update('configopts', '--with-MPI-include=%s' % os.getenv('MPI_INC_DIR'))
 
-        for dep in ["BLAS", "LAPACK"]:
-            libs = ' '.join(os.getenv('%s_STATIC_LIBS' % dep).split(','))
-            self.cfg.update('configopts', '--with-%s-libs="%s"' % (dep.lower(), libs))
-            self.cfg.update('configopts', '--with-%s-lib-dirs="%s"' % (dep.lower(),
-                                                                      os.getenv('%s_LIB_DIR' % dep)))
+        # Only supports external libraries when building a shared library.
+        self.cfg.update('configopts', '--enable-shared')
```

Comment:
@Micket does this mean that the static lib not is no longer built? if so, can't we make sure both are built (even though the static lib does not support external libs) ?

Reply:
I actually intentionally excluded this to make sure we didn't get a gimped version.

Reply:
The only easyconfigs we have that depend on `Hypre` are `PETSc`, so easy to test if the static `Hypre` lib is actually required...